static/js/components/Card.tsx

```diff
@@ -9,7 +9,7 @@ export default function Card(props: Props): JSX.Element {
 
   return (
     <div className="studio-card">
-      <div className="card-contents p-4">{children}</div>
```

Comment:
As far as I can tell, this class is unused

Reply:
ah yup, I think we copied over this component from another project and then refactored a bit and that class slipped through the net